lib/mix/tasks/compile/live_view.ex

```diff
@@ -0,0 +1,81 @@
+defmodule Mix.Tasks.Compile.LiveView do
```

Comment:
What if we call it `Compile.Heex`, as it is mostly to validate HEEx component?

Reply:
Unless we end up diverging on the core features, which I hope we don't 🙂, my intention is to make Surface (or any other templating solution) use the same API. We could still use `heex` as the name of the compiler but I have the feeling it might be confusing for users.

Reply:
I see. So LiveView is fine by me.